<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_dialog</class>
 <widget class="QDialog" name="user_dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 127);
font: 75 italic 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>个人中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btntime">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
   </property>
   <property name="text">
    <string>出发日期</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>248</width>
     <height>221</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">

QToolButton#qt_calendar_prevmonth,QToolButton#qt_calendar_nextmonth,QToolButton#qt_calendar_monthbutton,QToolButton#qt_calendar_yearbutton{
border:0px solid #000000;
border-radius:3px;
margin:3px 3px 3px 3px;
padding:3px;
background-color:rgba(0,0,0,0);

}
 
QToolButton#qt_calendar_prevmonth:hover,QToolButton#qt_calendar_nextmonth:hover,QToolButton#qt_calendar_monthbutton:hover,QToolButton#qt_calendar_yearbutton:hover,QToolButton#qt_calendar_prevmonth:pressed,QToolButton#qt_calendar_nextmonth:pressed,QToolButton#qt_calendar_monthbutton:pressed,QToolButton#qt_calendar_yearbutton:pressed{
border:1px solid #D8D8D8;

}
 
QCalendarWidget QSpinBox#qt_calendar_yearedit{
margin:3px 3px 3px 3px;
padding:0px -7px 0px 0px;
}
 
QDateEdit QCalendarWidget QSpinBox#qt_calendar_yearedit,QDateTimeEdit QCalendarWidget QSpinBox#qt_calendar_yearedit{
padding:0px -2px 0px 0px;
}
 
QCalendarWidget QToolButton::menu-indicator{

	image: none;
}
 
QCalendarWidget QTableView{
border-width:0px;
	
}
 
QCalendarWidget QWidget#qt_calendar_navigationbar{
border:1px solid #575757;
border-width:1px 1px 0px 1px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);

}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::NoHorizontalHeader</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
   <property name="dateEditAcceptDelay">
    <number>1500</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calenda">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认选择</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_serachcity">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-color: rgb(255, 170, 255);
font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
        </property>
        <property name="text">
         <string>按城市搜索</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_searchnum">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
      </property>
      <property name="text">
       <string>按航班号搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
   </property>
   <property name="text">
    <string>确认购买</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>89</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 3px;
</string>
   </property>
   <item>
    <property name="text">
     <string>CA1846</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MU5101 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MU5171</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HU7604</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CZ3908</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CZ9285</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CZ9274</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
   </property>
   <property name="text">
    <string>航班号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>60</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
   </property>
   <property name="text">
    <string>到达城市</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Microsoft New Tai Lue&quot;;</string>
   </property>
   <property name="text">
    <string>出发城市</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>89</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 3px;
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>89</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 3px;
</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_showticket">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>89</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
